Task 9/thesmallstar/visualise data ruppe-usd & time.m

```diff
@@ -0,0 +1,5 @@
+
+t = readtable('P:\EXINUS.csv');
```

Comment:
Using '`./` will be a much better choice here than hardcoding the path `P:\....`. Since in that way anybody cloning the repo can use it.